embed.fnc

```diff
@@ -6108,11 +6107,13 @@ Cp	|int	|do_spawn	|NN char *cmd
 Cp	|int	|do_spawn_nowait|NN char *cmd
 #endif /* defined(VMS) || defined(WIN32) */
 #if defined(WIN32)
+CTsdR	|void * |get_context
```

Comment:
I suspect the Win32 failures are because the Win32 Perl_get_context() isn't static.

Reply:
FWIW, I fixed that as part of the conflict resolution on embed.fnc